api/policy.go

```diff
@@ -27,10 +27,14 @@ const (
 	// PolicyTypeSQLReview is the sql review policy type.
 	PolicyTypeSQLReview PolicyType = "bb.policy.sql-review"
 
-	// PipelineApprovalValueManualNever means the pipeline will automatically be approved without user intervention.
+	// PipelineApprovalValueManualNever means the pipeline proceeds automatically if possible without the assignee's intervention, and the assignee is chosen from the workspace owners or DBAs.
 	PipelineApprovalValueManualNever PipelineApprovalValue = "MANUAL_APPROVAL_NEVER"
-	// PipelineApprovalValueManualAlways means the pipeline should be manually approved by user to proceed.
+	// PipelineApprovalValueManualNeverProjectOwner means the pipeline proceeds automatically if possible without the assignee's intervention, and the assignee is chosen from the project owners.
+	PipelineApprovalValueManualNeverProjectOwner PipelineApprovalValue = "MANUAL_APPROVAL_NEVER_PROJECT_OWNERS_ONLY"
+	// PipelineApprovalValueManualAlways means the pipeline should be manually approved by the assignee to proceed, and the assignee is chosen from the workspace owners or DBAs.
 	PipelineApprovalValueManualAlways PipelineApprovalValue = "MANUAL_APPROVAL_ALWAYS"
+	// PipelineApprovalValueManualAlwaysProjectOwner means the pipeline should be manually approved by the assignee to proceed, and the assignee is chosen from the project owners.
+	PipelineApprovalValueManualAlwaysProjectOwner PipelineApprovalValue = "MANUAL_APPROVAL_ALWAYS_PROJECT_OWNERS_ONLY"
```

Comment:
Variable name and value should somehow match. Either variable name = PipelineApprovalValueManualAlwaysProjectOwnerOnly, or MANUAL_APPROVAL_ALWAYS_PROJECT_OWNERS. I prefer the latter - brief.